x-pack/plugins/spaces/server/routes/api/v1/spaces.test.js

```diff
@@ -173,9 +262,11 @@ describe('Spaces API', () => {
   });
 
   test('POST space/{id}/select should respond with the new space location when a baseUrl is provided', async () => {
```

Comment:
nit: the name of this test (which this PR didn't change) kind of throws me off, because the `baseUrl` is being pulled from `server.config()` but the test description made me think it was being grabbed from the payload itself.